pkg/scheduler/core/generic_scheduler.go

```diff
@@ -115,6 +115,12 @@ func (g *genericScheduler) prioritizeClusters(
 		return result, err
 	}
 
+	if klog.V(4).Enabled() {
+		for plugin, nodeScoreList := range scoresMap {
+			klog.Infof("Plugin %s scores on %v/%v => %v", plugin, spec.Resource.Namespace, spec.Resource.Name, nodeScoreList)
```

Comment:
What's the log look like? Give an example here?

Reply:
Generally looks good. Just want to confirm the log here.

Reply:
the log format is following as
![image](https://user-images.githubusercontent.com/71266853/160226780-630b4480-4e72-4831-915a-f96cd0ddb322.png)